python/cudf/cudf/_lib/parquet.pyx

```diff
@@ -316,6 +320,74 @@ cpdef read_parquet(filepaths_or_buffers, columns=None, row_groups=None,
         df._data.label_dtype = cudf.dtype(column_index_type)
     return df
 
+cpdef read_parquet_metadata(filepaths_or_buffers):
+    """
+    Cython function to call into libcudf API, see `read_parquet_metadata`.
+
+    filters, if not None, should be an Expression that evaluates to a
```

Comment:
I don’t see filters here. Is this comment a copy-paste error?

Reply:
Yep, copy paste error. Thanks for catching this!